<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TakeLineWidget</class>
 <widget class="QWidget" name="TakeLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="placeholderText">
      <string>arguments 1 : filename or string with '\n'</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="removelineEdit">
     <property name="placeholderText">
      <string>arguments 2 : line number(it starts 0)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="returnLineEdit">
     <property name="placeholderText">
      <string>arguments 3 : return local variant name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
